conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
 - cuda-version=12.9
 - cupy>=13.6.0
 - cxx-compiler
-- cython>=3.0.3
+- cython>=3.0.0,<3.2.0
```

Comment:
For example think we want to keep the lower bound. And also add alpha here

These are both captured in the other PR

Though realize some of the fixes here are going beyond a generated pinning of dependencies

Hence why it would be help to combine them in one PR and going with yours is ok with me